suite-native/device-manager/src/components/WalletList.tsx

```diff
@@ -1,20 +1,59 @@
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
 
-import { selectDeviceInstances } from '@suite-common/wallet-core';
+import {
+    createDeviceInstanceThunk,
+    selectDevice,
+    selectDeviceInstances,
+    selectDeviceState,
+    selectIsPortfolioTrackerDevice,
+} from '@suite-common/wallet-core';
 import { VStack } from '@suite-native/atoms';
 import { TrezorDevice } from '@suite-common/suite-types';
 
 import { WalletItem } from './WalletItem';
+import { selectHasNonPassphraseDevice } from '@suite-native/device/src/selectors';
 
 type WalletListProps = {
     onSelectDevice: (device: TrezorDevice) => void;
 };
 
 export const WalletList = ({ onSelectDevice }: WalletListProps) => {
+    const dispatch = useDispatch();
     const devices = useSelector(selectDeviceInstances);
+    const selectedDevice = useSelector(selectDevice);
+    const hasNonPassphraseDevice = useSelector(selectHasNonPassphraseDevice);
+    const deviceState = useSelector(selectDeviceState);
+    const isPortfolioTrackerDevice = useSelector(selectIsPortfolioTrackerDevice);
+
+    const isSelectable = devices.length > 1 || !hasNonPassphraseDevice;
```

Comment:
Nit: You are negating already negated boolean value `!hasNonPassphraseDevice` and you are using it the same way few lines below. What about changing it to `hasPassphraseDevice`? The readability is better. You would have to change the selector, though.